src/allowlist/src/main.rs

```diff
@@ -13,11 +13,19 @@ use axum::{
 };
 use bb8::{Pool, PooledConnection, RunError};
 use bb8_redis::RedisConnectionManager;
+use futures::future::FutureExt;
 use move_core_types::account_address::{AccountAddress, AccountAddressParseError};
 use redis::{AsyncCommands, RedisError};
 use serde::Serialize;
+use std::time::Duration;
+use tokio::signal;
+use tower_http::timeout::TimeoutLayer;
+use tower_http::trace::TraceLayer;
 use tracing::info;
 
+/// The timeout period for pending requests.
+const PENDING_REQUEST_TIMEOUT: u64 = 10;
```

Comment:
This is confusing.

I'd rather have:

```suggestion
const PENDING_REQUEST_TIMEOUT: Duration = Duration::from_secs(10);
```

You do this anyway down below. Better put it here for better readability IMO.

Reply:
Addressed in ece33d7